scripts/run_tests

```diff
@@ -4,8 +4,8 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-# For each Rust workspace, first run tests, then run doc tests, then run clippy (turning warnings
-# into errors).
+# For each Rust workspace, first build wasm modules, then run tests, then
```

Comment:
Perhaps  this?

~~~
    7 # For each Rust workspace, first build Wasm modules where appropriate, then run tests, then
    8 # run doc tests, then run clippy (turning warnings into errors).
~~~

Also added `TODO(#472)`, and I'll revisit this when rebasing as I think we're moving to a single workspace

Reply:
(It just seemed odd to say "first build Wasm modules where appropriate, then run tests" when these isn't an explicit `cargo build` step)

Reply:
There is an additional step for the examples now `cargo build --target wasm32-unknown-unknown --manifest-path=./examples/Cargo.toml` but only for the examples workspace. I haven't checked master yet but I'm also presuming this will change when I rebase as we are moving to a single workspace.